apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ChatView.kt

```diff
@@ -1345,6 +1348,24 @@ fun BoxWithConstraintsScope.FloatingButtons(
         firstItemIsVisible = firstVisibleIndex == 0
       }
   }
+  LaunchedEffect(listState) {
```

Comment:
Make sure that you really need to put listState here. Isn't `LaunchedEffect(Unit)` is enough?